### PR TITLE
Linuxkit integration

### PR DESCRIPTION
**Pull request by giggsoff**
Integration of linuxkit
Sub-commands to work with GCP via linuxkit

### CODE REVIEW

**Comment by rvs** [comment]
Haven't had a chance to review in details yet, but if any code was taken from linuxkit project -- that needs to be very explicitly mentioned before this can be merged.

File: README.md [hunk 1]
```diff
@@ -266,12 +266,14 @@ The current sub-commands are:
...
-`eden utils certs` - generate certificates for Adam and EVE
-`eden utils download eve` - download EVE live image from docker hub
-`eden utils download eve-rootfs` - download EVE rootfs image from docker hub
-`eden utils sd` - get information about EVE from provided SD card
+   * `eden utils certs` - generate certificates for Adam and EVE
+   * `eden utils download eve` - download EVE live image from docker hub
+   * `eden utils download eve-rootfs` - download EVE rootfs image from docker hub
+   * `eden utils sd` - get information about EVE from provided SD card
+   * `eden utils gcp` - sub-commands to work with Google Cloud Platform
+
```
Comment (rvs) [inline README.md:279]: @giggsoff @mydatascience   can we please separate unrelated changes into separate PRs? This would really help with reviewing your changes much more quickly

**Comment by rvs** [comment]
Hey @mydatascience -- just please update LICENSE and NOTICE as it is done in EVE itself re: Linuxkit

**Comment by mydatascience** [comment]
> Hey @mydatascience -- just please update LICENSE and NOTICE as it is done in EVE itself re: Linuxkit

Is it ok now @rvs ?
